string_assignment.js

```diff
@@ -0,0 +1,133 @@
+
+// String Exercises
```

Comment:
Overall good demonstration of string methods and properties. Some problems could be solved better as feedback below suggests. Keep practicing logics as it is one of the most important aspect of coding. Keep it up!